rviz_common/src/rviz_common/render_panel.cpp

```diff
@@ -296,7 +296,7 @@ void RenderPanel::resizeEvent(QResizeEvent * event)
 
 const Ogre::Vector3 RenderPanel::default_camera_pose_ = Ogre::Vector3(999999, 999999, 999999);
 
-#if 0
+// TODO(jacobperron): Is there something to do related to the render window?
```

Comment:
If you didn't need to and it behaves like it does in rviz in ROS 1, then you can remove this todo.

Reply:
69ee286